src/frontend/src/.ic-assets.json

```diff
@@ -2,7 +2,7 @@
   {
     "match": "**/*",
     "headers": {
-      "Content-Security-Policy": "default-src 'self'; script-src 'self' 'unsafe-hashes' 'sha256-MhtPZXr7+LpJUY5qtMutB+qWfQtMaPccfe7QXtCcEYc=' 'wasm-unsafe-eval'; connect-src 'self' https://icp-api.io; img-src 'self' data:; style-src * 'unsafe-inline'; style-src-elem * 'unsafe-inline'; font-src 'self' data:; object-src 'none'; base-uri 'self'; frame-ancestors 'none'; form-action 'self'; upgrade-insecure-requests;",
+      "Content-Security-Policy": "default-src 'self'; connect-src 'self' https://icp-api.io; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval'; form-action 'self'; object-src 'none'; frame-ancestors 'none'; upgrade-insecure-requests; block-all-mixed-content",
```

Comment:
Unrelated change, but the UI keeps breaking because of the hash not being updated. Later on I wanna come back to this and generate this hash at build time or something.

The other deleted directives are just redundant.